packages/shared/src/components/profile/Header.tsx

```diff
@@ -97,6 +100,27 @@ export function Header({
         />
         {!isSameUser && (
           <CustomFeedOptionsMenu
+            onAdd={(feedId) =>
+              follow({
+                id: user.id,
+                entity: ContentPreferenceType.User,
+                entityName: user.username,
+                feedId,
+              })
+            }
+            onUndo={(feedId) =>
+              unfollow({
+                id: user.id,
+                entity: ContentPreferenceType.User,
+                entityName: user.username,
+                feedId,
+              })
+            }
+            onCreateNewFeed={() =>
+              router.push(
+                `/feeds/new?entityId=${user.id}&entityType=${ContentPreferenceType.User}`,
```

Comment:
Why do we pass the extra stuff?

Reply:
so onSuccess after creating custom feed we can follow the entity automatically (happens in bookmarks as well)